aviary/variable_info/enums.py

```diff
@@ -195,3 +195,7 @@ class Verbosity(Enum):
 
     def __str__(self):
         return str(self.value)
+
+    @classmethod
+    def set(cls):
```

Comment:
I think `values` would be a better name than `set` for this method.

Reply:
agree. done.